### DISCUSSION TITLE
Export & Sync

### DISCUSSION THREAD

**Discussion by vasnt** [twentyhq/twenty#5891] [Ideas]
1) How to backup/Export twenty database/tables
2) How to sync Twenty with nocodb/teable/other tabular database?

**Comment by ReessKennedy** [comment]
Contemplating sync with Nocodb or Baserow or Airtable -- one reason being that field manipulation and ability to bulk update fields is easier and faster in these tool.

**Comment by BOHEUS** [comment]
For doing backup of Twenty, there are instructions in docs how to do it https://docs.twenty.com/developers/self-host/capabilities/upgrade-guide#general-guidelines

Regarding sync, you can create an app which will synchronize with other databases (more info here https://docs.twenty.com/developers/extend/apps/getting-started) but I'd wait till SDK is mature and v1.0 is released (soon this year)
